src/removeLastHangulCharacter.spec.ts

```diff
@@ -0,0 +1,15 @@
+import { describe, it, expect } from 'vitest';
+import { removeLastHangulCharacter } from './removeLastHangulCharacter';
+
+describe('removeLastCharacter', () => {
```

Comment:
마이너합니다! 파일명, 함수명과 describe를 맞춰도 좋을 것 같아서요!

```suggestion
describe('removeLastHangulCharacter', () => {
```

Reply:
오 바로 반영하도록 하겠습니다!

Reply:
https://github.com/toss/es-hangul/pull/56/commits/a72646e7eb6fe286c77681dca001ebb7e4d51ef8 에서 반영 완료하였습니다 🙇